Cargo.toml

```diff
@@ -8,7 +8,7 @@ authors = ["Gavin Peacock <gpeacock@adobe.com"]
 crate-type = ["lib", "cdylib"]
 
 [dependencies]
-c2pa = { version = "0.44.0", features = [
+c2pa = { version= "0.45.0", features = [
```

Comment:
Can we keep the spacing consistent here?

Reply:
Done.